src/main/java/co/rsk/federate/FedNodeRunner.java

```diff
@@ -138,7 +138,7 @@ public void run() throws Exception {
             int hsmVersion = protocol.getVersion();
```

Comment:
The idea would be to have something like this:

```suggestion
            SignerVersion hsmVersion = SignerVersion.valueOf(protocol.getVersion());
```

Maybe to change `protocol.getVersion` to return the enum directly would be better.

Reply:
Done

Reply:
I dont think you pushed your changes to the branch

Reply:
Oh, sorry, you're right; I didn't add this change. The thing is, I think this change should be part of the second refactor where the use of the hsmVersion as `int` is replaced for the enum. So, I decided that it would be better to do that change altogether with the big refactor. Wdyt?

Reply:
Sounds good :)

Reply:
Did you create a task for this already?

Reply:
Done